app/providers.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+
+import * as Sentry from "@sentry/nextjs";
+import posthog from "posthog-js";
+import { PostHogProvider } from "posthog-js/react";
+import { useEffect } from "react";
+
+export function Analytics({
```

Comment:
Please tell me why do you've changed the name from example.

https://posthog.com/docs/libraries/next-js

<img width="722" alt="image" src="https://github.com/user-attachments/assets/cc72ed82-691d-4ffb-92b6-5d6ca0841fbf">

Reply:
This is a remnant of creating a nested function that included PHProvider() in the early implementation stage. For now, PHProvider is appropriate as it does not need to be this name.